src/pages/privacy-page.vue

```diff
@@ -1,5 +1,8 @@
 <template>
   <v-container>
+    <router-link to="/">
+      <v-btn @click="to = '/'" rounded color="primary" depressed dark>Zurück</v-btn>
+    </router-link>
```

Comment:
Hier kann das `<router-link>` ganz weg und das `:to="..:"` direkt an die `<v-btn>`-Komponente dran, weil `<v-btn>` auch ein `<router-link>` sein kann.

Reply:
done